node_modules/oae-core/groupprofile/groupprofile.html

```diff
@@ -0,0 +1,42 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/groupprofile.css" />
+
+<div class="groupprofile-widget">
+    <div id="groupprofile-header-container"><!-- --></div>
+    <div id="groupprofile-description-container"><!-- --></div>
+    <div id="groupprofile-members-container"><!-- --></div>
+</div>
+
+<div id="groupprofile-header-template"><!--
+    ${header('__MSG__ABOUT__', action)}
+--></div>
+
+<div id="groupprofile-description-template"><!--
+    {if contextProfile.description}
+        <div class="groupprofile-description">${oae.api.util.security().encodeForHTMLWithLinks(contextProfile.description)}</div>
+    {elseif contextProfile.isManager}
+        <div class="groupprofile-description-empty">
+            <button class="btn btn-link group-trigger-editgroup" data-editgroup-focus="#editgroup-description">
+                __MSG__ADD_A_DESCRIPTION_TO_THIS_GROUP__
```

Comment:
The text in this button should jump to multiple lines when necessary:

![screen shot 2015-04-20 at 19 51 45](https://cloud.githubusercontent.com/assets/109850/7244883/f8829b3e-e796-11e4-8dd2-9a27b7884df6.png)